src/chapters/exceptions/exceptions-control-flow.rst

```diff
@@ -0,0 +1,72 @@
+Exceptions-Control-Flow
+=======================
+
+In some instances we receive input from a user, and we can't always ensure they will type in something that will run perfectly with our code.
+
+JavaScript gives us some control flow tools to handle exceptions, by catching thrown exceptions.
+
+When we are anticipating an exception, and we want our program to continue running, and not experience the runtime error we can tell JavaScript to **try** to run a block of code, and if a specific exception is thrown to **catch** the exception and to run a different block of code.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+      :caption: Code
+
+      try {
+          console.log(x[0]);
+      }
+      catch(ReferenceError) {
+          console.log("We caught a ReferenceError, but our program continues to run!");
+          console.log("0");
+      }
+
+   .. sourcecode:: js
+      :caption: Output
+
+      We caught a ReferneceError, but our program continues to run!
+      0
+
+   In this example we attept to print out the first element of our variable x. We forgot to define x, but nested our code inside of a *try* statement, and specifically **catch** ReferenceErrors. It should be noted that only ReferenceErrors will be caught by the catch statement. However, we can define as many catch statements as we want.
+
+JavaScript also provides us with a **finally** statement, a block of code that always runs every time regardless to if the try, or catch blocks run.
```

Comment:
``finally``

"statement: a block of code ... regardless if the ``try`` or ``catch`` blocks run."

Reply:
I used double ` before finally, try, and catch. Is that correct? Also removed the unnecessary "to".

Reply:
fixed after talking to Sally